internal/module/services/version_manager/version_manager.go

```diff
@@ -67,6 +72,16 @@ func (s *VersionManagerService) RegisterWithServer(grpcServer grpc.ServiceRegist
 
 	services.RegisterVersionManagerServer(grpcServer, s)
 
+	// Create release service struct.
+	rs := &release.ReleaseService{
+		Db:     s.Db,
+		Table:  release.ReleaseTableName,
+		Schema: release.GetReleaseSchema(release.ReleaseTableName),
+	}
+
+	// Register with publisher.
+	releaseSvc.RegisterPublisherServer(grpcServer, rs)
```

Comment:
As Vesna mentioned we should not be doing this here.  Instead we should adding an argument to the version manager command to take a release service URL which we will reference when we publish the release in `PublishVersion`